portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,102 @@
+        <p>For my senior project in high school, I partnered with InnovationV and designed a website for one of
+           their clients.
+        </p>
+        <hr>
+
+        <h2>
+            <cursive>Places I Would Love to Visit</cursive>
+        </h2>
+        <div class="gallery">
+            <img src="https://specials-images.forbesimg.com/imageserve/1182337590/960x0.jpg?fit=scale" alt="Bogota, Colombia" width="350" height="250">
+            <div class="desc">Bogota, Colombia</div>
+        </div>
+
+        <div class="gallery">
+            <img src="https://ceoworld.biz/wp-content/uploads/2019/11/Venice-Italy-1.jpg" alt="Venice, Italy" width="350" height="250">
+            <div class="desc">Venice, Italy</div>
+        </div>
+
+        <div class="gallery">
+            <img src="https://cdn.britannica.com/36/191736-050-5726A7F4/view-Lagos-Nigeria.jpg" alt="Lagos, Nigeria" width="350" height="250">
+            <div class="desc">Lagos, Nigeria</div>
+        </div>
+
+        <div class="gallery">
+            <img src="https://afar-production.imgix.net/uploads/syndication/holland_americas/images/Z2NcYxB3NT/original_BARCELONA_HERO__ESY-014512839.jpg?w=750&h=563&fit=crop" alt="Barcelona, Spain" width="350" height="250">
+            <div class="desc">Barcelona, Spain</div>
+        </div>
+
+        <div class="gallery">
+            <img src="https://www.earthquakeauthority.com/getattachment/a14542f8-28c3-4402-b636-d95c6fe79fd0/los-angeles-earthquake-prediction-and-risk?maxsidesize=1200&width=1200&height=500" alt="Los Angeles, California" width="350" height="250">
+            <div class="desc">Los Angeles, California</div>
+        </div>
+        <div class="gallery">
+            <img src="https://image-cdn.hypb.st/https%3A%2F%2Fhypebeast.com%2Fimage%2F2019%2F10%2Ftokyo-best-big-city-visit-2019-01.jpg?q=90&w=1400&cbr=1&fit=max" alt="Tokyo, Japan" width="350" height="250">
+            <div class="desc">Tokyo, Japan</div>
+        </div>
+        <p>I would love to travel to these cities (and many more) to experience its culture. I personally believe that my life would not be complete withoug immersing myself in places outside my comfort zone. In addition to my love for traveling, I also enjoy taking photos of people, cities, and nature. Capturing a landscape is one of the most beautiful things. Capturing a busy city is even more captivating. I spent my previous summer in New York City, and some of the photos of the city are some of my favourites.</p>
+        <hr>
+        <h3>
+            <cursive>My Hidden Talent</cursive>
+        </h3>
+        <p>Although I write with my left hand, I am actually ambidextrous. I do many activities with both hands and can even write with both hands (not very well, but its legible)!</p>
+        <h4>
+            <cursive>Connect with me on <a href=https://www.linkedin.com/in/sydney-beasley-6b04b8168>LinkedIn</a> and <a href=https://www.github.com/Sydney-18>GitHub</a></cursive>
```

Comment:
This sentence doesn't look like it's semantically a heading, so I'd suggest not using `h4`. Instead, you could wrap it in a `p` with a special class, like "note" or "sign-off" or something else descriptive, then style that the way you want it to look in CSS.